pylintrc

```diff
@@ -62,6 +62,7 @@ confidence=
 # --disable=W".
 disable=missing-class-docstring,
         unsubscriptable-object, # Pylint does not support Genric class see https://github.com/PyCQA/pylint/issues/3520
+        too-many-lines,
```

Comment:
I think these exceptions should be put to the specific files as they give us a good warning we should be cleaning up.